students/donggyu/users/views.py

```diff
@@ -52,10 +50,14 @@ def post(self, request):
             email    = data['email']
             password = data['password']
 
-            if User.objects.filter(email=email, password=password).exists():
-                return JsonResponse({"message": "INVALID_USER"}, status = 401)
+            if not bcrypt.checkpw(password.encode('utf-8'), User.objects.get(email=email).password.encode('utf-8')):
```

Comment:
동규님~! 반복적으로 작성되는 코드는 `user = User.objects.get(email=email)`을 통해 선언한 뒤 56번 line에도 적용해주세요.
```suggestion
            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
```

Reply:
수정 완료되었습니다.